spec/system/admin/poll/polls_spec.rb

```diff
@@ -33,6 +33,21 @@
     expect(page).not_to have_content "There are no polls"
   end
 
+  scenario "Can see all polls despite it is with the published attribute set to true or not" do
+    create(:poll)
```

Comment:
What do you think about giving theses polls a name and use it to make the assertion? There is a good example in the line `12`, "Index show polls list order by starts at date".

Reply:
I'll give it a try.

Reply:
Since this test is only to check the poll's quantity and not order, names aren't needed.